python/tank/platform/util.py

```diff
@@ -261,3 +261,33 @@ def get_logger(module_name):
     return logging.getLogger(full_log_path)
 
 
+def get_hyperlink_html(url, name, color=None, bold=True):
```

Comment:
wondering if this could this be part of the engine class? That way each DCC integration could potentially override it with a custom implementation. This way we wouldn't have to pass stuff like the color. Thinking about what @robblau said yesterday that apps shouldn't encode too much stuff in terms of styling. Here, we are giving apps an option to explicitly override the engine scheme. For the shotgun panel, we are passing in `self._app.style_constants["SG_HIGHLIGHT_COLOR"],` which seems like we are looping round - what if it was instead simple `engine.get_hyperlink_html("https://foo.bar")`?